sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/SQLQuerySuite.scala

```diff
@@ -2040,4 +2040,83 @@ class SQLQuerySuite extends QueryTest with SQLTestUtils with TestHiveSingleton {
       assert(setOfPath.size() == pathSizeToDeleteOnExit)
     }
   }
+
+  test("insert overwrite to dir from hive metastore table") {
```

Comment:
Move all these test cases to `InsertIntoHiveTableSuite`. Also we should rename that suite to `InsertSuite`